main/src/main/java/org/mobilitydata/gtfsvalidator/report/model/ReportSummary.java

```diff
@@ -45,7 +42,7 @@ public ReportSummary(NoticeContainer container, VersionInfo versionInfo) {
             .collect(
                 Collectors.groupingBy(
                     NoticeView::getSeverityLevel,
-                    LinkedHashMap::new,
+                    () -> new TreeMap<>(Comparator.comparingInt(SeverityLevel::getOrder)),
```

Comment:
My understanding is the Enums already implements comparable having the natural order the order of the constant declaration. As the declaration order is INFO, WARNING and ERROR, we can use the reserve order to get the desired sorting, can you try here?
 ```suggestion
                    () -> new TreeMap<>(Comparator.reserveOrder()),
```

Reply:
it works ✅